code/src/pages/movieDetails.css

```diff
@@ -0,0 +1,106 @@
+.details-back-arrow {
+    display: inline-flex;
+    width: 40px;
+}
+
+.details-text-arrow {
+    display: inline-flex;
+    align-self: center;
+    margin-left: 10px;
+    color: white;
+    font-weight: 700;
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+    transition: .8s;
+}
+
+.details-text-arrow:hover {
+    margin-left: 20px;   
+}
+
+.back-link {
+    position: absolute;
+    top: 10px;
+    right: 0px;
+    left: 0px;
+    padding: 50px;
+    text-decoration: none;
+    display: inline-flex;
+}
+
+.details-container {
+    display: flex;
+    flex-direction: row;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    background-color: black;
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    border: none;
+    align-items: flex-end;
+}
+
+.details-summary img {
+    border: 3px solid white;
+    width: 300px;  
+    display: none; 
+}
+
+.details-summary {
+    display: inline-flex;
+    align-items: flex-end;
+    color: white;    
+}
+
+.details-text {
+    margin: 15px;
+    max-width: 400px;
+    background: rgb(8, 8, 8, 0.565);
+    padding: 20px;
+}
+
+.details-text h1 {
+    font-size: 24px;
+    font-weight: 700;
+}
+
+.details-text h2 {
+    font-size: 17px;
+    font-weight: 400;
+}
+
+.details-rating {
+    color: red;
+    font-size: 17px;
+    margin-left: 8px;
+}
+
+@media (min-width: 667px) {
```

Comment:
Some repetition of css in both desktop view and in the media query. Maybe some of the styling in the media queries could be removed, like for example in line 89-90 and 97 and 98?